### PR TITLE
vSphere Document Permissions Update

### PR DESCRIPTION
**Pull request by codekow**
The documented permissions for vSphere can use some updated details

### CODE REVIEW

**Comment by jstuever** [comment]
/cc @jcpowermac
Can you give this a quick review?

File: docs/user/vsphere/privileges.md [hunk 1]
```diff
@@ -35,28 +35,32 @@ The easiest way to ensure proper permissions is to grant Global Permissions to t
...
+openshift-vcenter-level | False | vCenter
+ReadOnly | False | Datacenter
+openshift-cluster-level | True | Cluster
+openshift-datastore-level | False | Datastore
+ReadOnly | False | Switch
+openshift-portgroup-level | False | Port Group
+ReadOnly | True | Virtual Machine folder (Top Level)
+openshift-folder-level | True | Virtual Machine folder
+
+In a cascading folder organization you will need  "`Read-only`" permissions with "`Propagate to children`" from the top folder level.
```
Comment (jcpowermac) [inline docs/user/vsphere/privileges.md:51]: @codekow not quite sure `Virtual Machine folder (Top Level)` is trying to convey. Do you mean if I have a nested folder structure the top-level of those folders? 

The _real_ top level `/datacenter/vm` is not displayed within vCenter.

Comment (codekow) [reply docs/user/vsphere/privileges.md:51]: Yes, specifically I am talking about a nested folder structure. This is specific to the user view and not the internal view. @jcpowermac

**Pull request closed without merging**